kong/plugins/request-transformer/migrations/postgres.lua

```diff
@@ -0,0 +1,64 @@
+return {
+  {
+    name = "2016-05-04-160000_req_trans_schema_changes",
+    up = function(_, _, factory)
+
+      local plugins, err = factory.plugins:find_all {name = "request-transformer"}
+      if err then
+        return err
+      end
+
+      for _, plugin in ipairs(plugins) do
+        for _, action in ipairs {"remove", "add", "append", "replace"} do
```

Comment:
Sorry for the late reply @thibaultcha , somehow missed this update. 

Is this what you mean? https://github.com/Mashape/kong/pull/1907/commits/833ec495c0ec68b3b64cbd7372b97d6f51b0cb23